obs_img_utils/utils.py

```diff
@@ -511,3 +511,40 @@ def filter_packages_by_name(packages_metadata, package_name):
         matching_packages[name] = packages_metadata[name]
 
     return matching_packages
+
+
+def get_condition_list_from_file(filename, logger):
+    """
+    Creates the conditions objects from a file with json dump.
+    """
+    with open(filename, 'r') as file:
+        data = json.load(file)
+
+    if type(data) != list:
+        msg = f'Loaded conditions from {filename} but not '
+        msg += 'in list format! Converting it...'
+
+        logger.warning(msg)
```

Comment:
Since message is only used once there's no advantage to create a new variable:

```
logger.warning(
    f'Loaded conditions from {filename} but not '
    'in list format! Converting it...'
)
```

The above works and also removes the extra string concatenation.

Reply:
Fixed!